modules/bridge/src/main/ts/ephox/bridge/components/dialog/HtmlPanel.ts

```diff
@@ -1,25 +1,31 @@
 import { FieldSchema, StructureSchema } from '@ephox/boulder';
-import { Result } from '@ephox/katamari';
+import { Fun, Result } from '@ephox/katamari';
 
 import * as ComponentSchema from '../../core/ComponentSchema';
 
 export interface HtmlPanelSpec {
   type: 'htmlpanel';
-  html: string;
+  html?: string;
```

Comment:
Because when you use the raw onInit then the html would be useless. I guess I could jus set it to an empty string in the math plugin to and then this wouldn't be optional.

Reply:
I see, if we want to use `onInit` also to create an HTML from scratch it make sense to have it optional

Reply:
I rolled back that change since this make the change lesser in scope.

Reply:
It kind of mixed since if you don't use the `onInit` function then html should be required since then it makes no sense. So I guess enforcing it would make sense since it just fills the inner HTML of the element and the onInit just passed that element then if that adds stuff or just adds a class or events or what ever it's up to the implementation.